### PR TITLE
Add audio parameter UI

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    private int width;
+    private int height;
+    private String name;
+
+    ResolutionVideo(String name,int width,int height ){
+        this.height=height;
+        this.width=width;
+        this.name=name;
+    }
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java:21]: I think that is missing javadoc for the methods

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- *
+ * <p>
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: Delete this

File: src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package org.fundacion.jala.converter.view.converter;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java:9]: Copyright is missing and author

File: src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+public class FrameVideo {
+    private String name="F";
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java]: Don't forget the spaces between "="

File: src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    private int frame;
+
+    public FrameVideo(int frame) {
+        this.name = name;
+        this.frame = frame;
+    }
+
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java:19]: Javadoc is missing.

File: src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        return frame;
+    }
+
+    public void setFrame(int frame) {
+        this.frame = frame;
+    }
+
+    @Override
+    public String toString() {
+        return frame+name;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java]: Don't forget the space between "+"

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java [hunk 7]
```diff
@@ -1,6 +1,6 @@
 /**
  * Copyright (c) 2021 Fundacion Jala.
- *
+ * <p>
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: Delete this, and don't forget to add author :)

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java [hunk 8]
```diff
@@ -11,17 +11,14 @@
...
 
-import javax.swing.JPanel;
-import javax.swing.JComboBox;
-import javax.swing.JCheckBox;
+import javax.swing.*;
 
-import java.awt.Font;
-import java.awt.Dimension;
-import java.awt.GridLayout;
+import java.awt.*;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: Specify the import library, change the "*"

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java [hunk 9]
```diff
@@ -11,17 +11,14 @@
 
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
 
-import javax.swing.JPanel;
-import javax.swing.JComboBox;
-import javax.swing.JCheckBox;
+import javax.swing.*;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: Specify the library, change the "*"

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java [hunk 10]
```diff
@@ -58,51 +53,71 @@ protected OutputSettings() {
...
+
+        resolutionComboBox = new JComboBox<ResolutionVideo>(
+                new ResolutionVideo[]{
+                        new ResolutionVideo("720p(HD)", 1280, 720),
+                        new ResolutionVideo("1920p", 1080, 1920),
+                        new ResolutionVideo("480p", 854, 480),
+                        new ResolutionVideo("240p", 426, 240),
+                        new ResolutionVideo("DVD", 720, 567),
+                        new ResolutionVideo("TV", 640, 480),
+                        new ResolutionVideo("Mobile", 320, 240)}
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java:66]: Check the indexing, it seems that it has to much space

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java [hunk 11]
```diff
@@ -58,51 +53,71 @@ protected OutputSettings() {
...
-        framesSelect.addItem("30F");
-        framesSelect.addItem("60F");
+        framesSelect=new JComboBox<FrameVideo>(
+                new FrameVideo[]{
+                        new FrameVideo(21),
+                        new FrameVideo(24),
+                        new FrameVideo(27),
+                        new FrameVideo(29),
+                        new FrameVideo(30),
+                        new FrameVideo(60)});
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java:82]: Same here, it seems that it has to much space

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java [hunk 12]
```diff
@@ -58,51 +53,71 @@ protected OutputSettings() {
...
     /**
      * Method that get selected frame
      * for video converter.
+     *
      * @return String, option selected of Frame.
      */
-    protected String getFrame() {
-        return framesSelect.getSelectedItem().toString();
+    protected int getFrame() {
+        FrameVideo frameVideo=(FrameVideo)framesSelect.getSelectedItem();
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: Don't forget to add spaces "="

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package org.fundacion.jala.converter.view.converter;
+import org.fundacion.jala.converter.view.utilities.JLabelStyle;
+import javax.swing.*;
+import java.awt.*;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java]: Specify the library, change the "*"

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+        add(audioChannelSelect);
+        add(metadataOption);
+    }
+
+    /**
+     * Method that set all possible volume
+     * for Audio converter.
+     */
+    protected void setVolumeSelect() {
+        volumeSelect.addItem(" ");
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java:63]: Don't know if is recommended to set an empty option, maybe can put some element as default?

File: src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java [hunk 15]
```diff
@@ -0,0 +1,42 @@
+package org.fundacion.jala.converter.view.converter;
+
+public class ResolutionVideo {
+    private int width;
+    private int height;
+    private String name;
+
+    ResolutionVideo(String name,int width,int height ){
+        this.height=height;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java]: Spaces between "="

File: src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @Override
+    public String toString() {
+        return  width+"X"+height+"     "+name;
```
Comment (Paola-Xaq) [inline src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java]: Add spaces before an after "+"

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java (hunk 2 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: solved

File: src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java (hunk 5 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java:19]: I forgot it, thanks

File: src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java (hunk 6 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/FrameVideo.java]: oks

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java (hunk 7 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: Oh no

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java (hunk 8 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java]: ok

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java (hunk 10 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java:66]: ok

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java (hunk 11 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/OutputSettings.java:82]: ok

**Review by jpgonzales27 — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java (hunk 14 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java:63]: Maybe de default is empty

**Review by Paola-Xaq — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java (hunk 13 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java]: ok

File: src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java (hunk 15 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java]: ok

File: src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java (hunk 1 above)
Comment (GustavoZHA) [reply src/main/java/org/fundacion/jala/converter/view/converter/ResolutionVideo.java:21]: ok

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**